app/assets/scripts/components/explore/prime-panel.js

```diff
@@ -156,6 +156,17 @@ function ExpMapPrimePanel (props) {
     }
   }, [currentZones?.data]);
 
+  React.useEffect(() => {
+    const availableZone = availableZoneTypes.filter(
+      (zoneType) =>
+        selectedResource !== RESOURCES.OFFSHORE || zoneType?.type !== BOUNDARIES
+    );
+    if (selectedResource === RESOURCES.OFFSHORE) {
+      availableZone.map(
```

Comment:
use find instead of map, it's more suitable for this case

```suggestion
     const targetZoneType = availableZone.find((zoneType) => selectedZoneType && selectedZoneType.size === '0' && zoneType.size === '25')
     
     if (targetZoneType) {
         setSelectedZoneType(targetZoneType);
     }
```